docs/source/api_ref_sparsity.rst

```diff
@@ -1,14 +0,0 @@
-.. _api_sparsity:
```

Comment:
This is one of the few things we do have docs for though, wondering what the root cause is still

Reply:
Ahhh I'm fixing that upstream https://github.com/pytorch/ao/pull/225

Reply:
We should probably still install dev-requirements.txt in the doc build for consistency?